apps/mobile/src/features/Settings/components/AppSettings/AppSettings.container.tsx

```diff
@@ -9,13 +9,26 @@ import { SafeFontIcon as Icon } from '@/src/components/SafeFontIcon/SafeFontIcon
 import { FloatingMenu } from '../FloatingMenu'
 import { LoadableSwitch } from '@/src/components/LoadableSwitch'
 import { useBiometrics } from '@/src/hooks/useBiometrics'
+import { useNotificationManager } from '@/src/hooks/useNotificationManager'
+import { useAppSelector } from '@/src/store/hooks'
+import { selectAppNotificationStatus } from '@/src/store/notificationsSlice'
 import { capitalize } from '@/src/utils/formatters'
 import { SAFE_WEB_FEEDBACK_URL } from '@/src/config/constants'
 
 export const AppSettingsContainer = () => {
-  const { toggleBiometrics, isBiometricsEnabled, isLoading, getBiometricsUIInfo } = useBiometrics()
+  const { toggleBiometrics, isBiometricsEnabled, isLoading: isBiometricsLoading, getBiometricsUIInfo } = useBiometrics()
+  const { enableNotification, disableNotification, isLoading: isNotificationsLoading } = useNotificationManager()
+  const isAppNotificationEnabled = useAppSelector(selectAppNotificationStatus)
   const { themePreference, setThemePreference } = useTheme()
 
+  const handleToggleNotifications = () => {
+    if (isAppNotificationEnabled) {
+      disableNotification()
+    } else {
+      enableNotification()
+    }
+  }
```

Comment:
This enables/disables the notifications globally in the app. 

If the user enables them, we try to subscribe automatically all safes for push notifications. 

If the user unsubscribes, then he is being automatically unsubscribed from all safes.